### ISSUE TITLE
Refactor Network Simulation

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/domain-ontology#74]
http://edamontology.org/operation_3562 (Network simulation) is now obsolete in EDAM. We need to decide how to refactor this.

_Originally posted by @allysonlister in https://github.com/FAIRsharing/domain-ontology/issues/73#issuecomment-769196835_

**Comment by allysonlister** [comment]
As EDAM states, "Notions of pathway and network were mixed up, EDAM 1.24 disentangles them." See http://www.ontobee.org/ontology/EDAM?iri=http%3A%2F%2Fedamontology.org%2Foperation_3562 for an alternate term. EDAM suggests Network analysis or Pathway analysis.

**Comment by allysonlister** [comment]
It looks like our best choice is to go with one of EDAM's suggestions, [Network analysis](https://www.ebi.ac.uk/ols/ontologies/edam/terms?iri=http%3A%2F%2Fedamontology.org%2Foperation_3927) http://edamontology.org/operation_3927

[12](https://fairsharing.org/search/?q=&content=standard&content=biodbcore&content=policy&content=collection&name=&taxonomies=&organisations=&shortname=&description=&supportlinks=&licenses=&countries=&maintainers=&expanded_onto_domains=network+simulation&expanded_onto_disciplines=&user_defined_tags=&search_state=visible&selected_facets=expanded_onto_domains_exact:Network%20simulation) records contain this domain, all of which can be reannotated with Network analysis. 

This term is not yet in DRAO and will need to be added. 

So, in summary:

- Add http://edamontology.org/operation_3927 to DRAO
-     remove http://edamontology.org/operation_3439 from DRAO.
-     update preferred label of operation_3927 from it's current EDAM value of "Network analysis" to "Biological network analysis" to fit EDAM's definition

Old IRI | New IRI | Note
------------ | ------------- | -------------
http://edamontology.org/operation_3562 | http://edamontology.org/operation_3927 | Deprecation of EDAM term resulted in modification of the domain tag that is being used within DRAO

@delphinedauga  and @Drosophilic please let me know your thoughts on this change. Thanks!

**Comment by Drosophilic** [comment]
Looks good. Thanks.
